cmd/cloudshell_open/main.go

```diff
@@ -228,14 +235,24 @@ func run(opts runOpts) error {
 	}
 
 	if err := waitForBilling(project, func(p string) error {
+		coupon, err := instrumentlessCoupon()
+		if err != nil {
+			return err
+		}
+
 		fmt.Print(errorPrefix+" "+
-			warningLabel.Sprint("GCP project you chose does not have an active billing account!")+
-			"\n  1. Visit "+linkLabel.Sprint(projectCreateURL),
-			"\n  2. Associate a billing account for project "+parameterLabel.Sprint(p),
+			warningLabel.Sprintf("Project '%s' does not have an active billing account!", project)+
```

Comment:
I think changing this text is bad for existing GCP users with an active billing account?

Reply:
You thinking we should make it clear they could use an existing billing account?  Like:

If they have one existing billing account:
```
Project 'foo' does not have an active billing account!
Would you like to use the existing `asdf` billing account with project `foo`? [Y/n]
n
Create a new billing account from a starter coupon:
https://console.cloud.google.com/usecoupon?asdf

Once you've completed creating the new billing account and associating it with your project, press [enter] to continue.
```

If they don't have an existing billing account, omit the option to use the existing one.

If they have more than one billing account, then:
```
Would you like to use an existing billing account with project `foo`? [Y/n]
Y
Select the billing account to use:
...
```

I'll see about doing the linking via the API.  Cause that'd make this nicer.

Reply:
Yeah that's what I was proposing.
Most users with billing accounts don't need instrumentless at all.

I think we send users to the Console page for linking because I don't think there is an API for it.